src/axom/primal/geometry/BoundingBox.hpp

```diff
@@ -70,40 +70,10 @@ struct ValueRange
   /*! \brief Returns the lowest representable value of type T */
```

Comment:
We should remove the ``ValueRange`` traits class here  and use ``std::numeric_limits`` directly instead.

Reply:
Done.